### PR TITLE
duck 16480

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: Library/Formula/duck.rb [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
+  url "https://dist.duck.sh/duck-src-4.6.2.16480.tar.gz"
+  sha1 "6e38a90c97bbe7d577791a4fbdd07872e8457101"
   head "https://svn.cyberduck.io/trunk/"
 
-  bottle do
-    cellar :any
-    sha1 "05353941783e34f1931b0cdd78a89f245d8daf33" => :yosemite
-    sha1 "a2120b7b4b7ae15f9b941c24bb9d4f7c36392e74" => :mavericks
-    sha1 "fb6e8e20d702cf3d9808e758328cd7ebc8830283" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/duck.rb]: Just leave this as-is and we'll update it when we pull, thanks.

Comment (dkocher) [inline Library/Formula/duck.rb]: I understood that the bottles will be newly added automatically by the merge?

Comment (mikemcquaid) [inline Library/Formula/duck.rb]: They will be but it helps the detection algorithm if they are still present in the formula. Thanks!

**Comment by dkocher** [comment]
Thanks for accepting the update quickly. Am I right that the pull request was cherry picked excluding the previous bottles? Can you clarify again if these are required for future updates or not.

**Comment by mikemcquaid** [comment]
We pulled the commit in this PR (32aae0c) and then the following commit will (acf1cefd2bdf14d16c8cec9d85336991dc52f8aa) always update the bottles. Future updates should be in the same format as this PR: just leave the `bottle do` block as-is. Thanks

**Comment by dkocher** [comment]
Thanks for the clarification.

**Pull request closed without merging**